src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -38,7 +40,8 @@ const RenderListFeed = (props) => {
     userId,
     onPressDownVote,
     selfUserId,
-    onPressUpvote
+    onPressUpvote,
+    offset
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `offset` prop is added here. This could be used for pagination or infinite scrolling purposes. However, without seeing how it's used in the rest of the code, it's hard to say if this is the best approach.
```diff
+     offset
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->